lib/ansible/modules/rpm_key.py

```diff
@@ -159,30 +162,38 @@ def fetch_key(self, url):
         tmpfile.close()
         return tmpname
 
-    def normalize_keyid(self, keyid):
+    def normalize_keyids(self, keyids):
         """Ensure a keyid doesn't have a leading 0x, has leading or trailing whitespace, and make sure is uppercase"""
-        ret = keyid.strip().upper()
-        if ret.startswith('0x'):
-            return ret[2:]
-        elif ret.startswith('0X'):
-            return ret[2:]
-        else:
-            return ret
+        normalized_keyids = []
+        for keyid in keyids:
+            ret = keyid.strip().upper()
+            if ret.startswith('0x'):
+                normalized_keyids.append(ret[2:])
+            elif ret.startswith('0X'):
+                normalized_keyids.append(ret[2:])
+            else:
+                normalized_keyids.append(ret)
```

Comment:
Can we maybe keep `normalize_keyid` and create a `normalize_keyids` that calls the singular version. That can make unit testing of this function in the future easier.